Documentation/Blazorise.Docs.Server/Startup.cs

```diff
@@ -33,14 +34,29 @@ public Startup( IConfiguration configuration )
     // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
     public void ConfigureServices( IServiceCollection services )
     {
-        services.AddRazorPages();
-        services.AddServerSideBlazor();
+        // Add services to the container.
+        services
+            .AddRazorComponents()
+            .AddInteractiveServerComponents();
 
```

Comment:
Want to try auto mode? By default docs opens in Server mode, but when dlls are available it opens in wasm.